mmdet/datasets/transforms/transforms.py

```diff
@@ -2217,13 +2217,13 @@ class Mosaic(BaseTransform):
     Modified Keys:
 
     - img
-    - img_shape
+    - img_scale
```

Comment:
```suggestion
    - img_shape
```

Reply:
You may have misunderstood, this field in docstring is to indicate which keys in result dictionary will be modified.

Reply:
Oh, I see, it is the result['img_shape']